repo/content/index/index_builder.go

```diff
@@ -63,8 +67,8 @@ func init() {
 // sortedContents returns the list of []Info sorted lexicographically using bucket sort
 // sorting is optimized based on the format of content IDs (optional single-character
 // alphanumeric prefix (0-9a-z), followed by hexadecimal digits (0-9a-f).
-func (b Builder) sortedContents() []Info {
-	var buckets [36 * 16][]Info
+func (b Builder) sortedContents() []BuilderItem {
```

Comment:
can we avoid `BuilderItem` by having `*Item` instead (assuming we can do `blob.ID` interning as suggested elsewhere)

Reply:
Yes, this comes down to whether we can avoid the introduction of `InfoCompact`, if we can use `Info` only, we will not need this `BuilderItem`